packages/sanity/src/ui/tab/Tab.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-restricted-imports */
+import {Tab as UITab, TabProps as UITabProps} from '@sanity/ui'
+import React, {forwardRef} from 'react'
+
+/**
+ * @internal
+ *
+ * Padding and font sizes are fixed in Studio UI <Tab> components.
+ */
+export type TabProps = Pick<
+  UITabProps,
+  'aria-controls' | 'focused' | 'icon' | 'id' | 'label' | 'selected' | 'tone'
```

Comment:
Re-added `icon` support for now, since this would be considered a breaking change (as users can configure custom icons for groups and document views)